content/2-templating/2-styling/aurelia1/styling.scss

```diff
@@ -0,0 +1,3 @@
+.title {
```

Comment:
This file is in ".scss" instead of ".css".
And it's not used in "styling.html" file. I doubt this file is executed.